### ISSUE TITLE
Can't append script tags inside `<svelte:head>`

### ISSUE THREAD

**Issue by buhodev** [inlang/inlang#1066]
## Problem

I am using (as recommended) the `autoModeWatcher` function from [Skeleton UI](https://www.skeleton.dev/utilities/lightswitches) to adjust the dark/light mode based on operating system preference. It is used like this inside of a Svelte component:

```svelte
<script>
  import { autoModeWatcher } from '@skeletonlabs/skeleton';
</script>

<svelte:head>{@html `<script>${autoModeWatcher.toString()} autoModeWatcher();</script>`}</svelte:head>
```

This appends (as a string) the body of the function inside the `head` tag of the HTML document and calls it. But just when I add that line to my project, it throws the following error:

```bash
[vite] Internal server error: Missing semicolon. (1:1)
  Plugin: vite-plugin-inlang-sdk-js-sveltekit
  File: C:/Users/buhodev/project/src/components/Header.svelte:1:1
  1  |  <script lang="ts">
     |   ^
  2  |          import { AppBar, LightSwitch } from '@skeletonlabs/skeleton';
  3  |          import HeaderLink from './HeaderLink.svelte';
```

If I remove the `svelte:head` line, then the error disappears.

```diff
- <svelte:head>{@html `<script>${autoModeWatcher.toString()} autoModeWatcher();</script>`}</svelte:head>
```

### Expected behavior

I can append

## Steps to reproduce

1. Install SvelteKit example and setup inlang
2. Install skeleton
3. Import the `autoModeWatcher` function from skeleton inside a root layout or component
4. See the error

### Additional information

- `inlang.config.js` file:

```js
export async function defineConfig(env) {
	// const { default: typesafeI18nPlugin } = await env.$import(
	// 	'https://cdn.jsdelivr.net/gh/ivanhofer/inlang-plugin-typesafe-i18n@2/dist/index.js'
	// );

	const { default: jsonPlugin } = await env.$import('https://cdn.jsdelivr.net/npm/@inlang/plugin-json@3/dist/index.js');

	const { default: standardLintRules } = await env.$import(
		'https://cdn.jsdelivr.net/npm/@inlang/plugin-standard-lint-rules@3/dist/index.js'
	);

	const { default: sdkPlugin } = await env.$import('https://cdn.jsdelivr.net/npm/@inlang/sdk-js-plugin@0.7.1/dist/index.js');

	// const { default: ideExtensionPlugin } = await env.$import(
	// 	'https://cdn.jsdelivr.net/npm/@inlang/ide-extension-plugin@latest/dist/index.js'
	// );

	return {
		referenceLanguage: 'en',
		plugins: [
			// typesafeI18nPlugin(),
			jsonPlugin({ pathPattern: './languages/{language}.json' }),
			standardLintRules(),
			sdkPlugin({
				languageNegotiation: {
					strategies: [{ type: 'url' }]
				}
			})
			// ideExtensionPlugin()
		]
	};
}
```

- `npx @inlang/cli config validate` output:

```sh
i 🔎 Validating the config file...
√ 🎉 Inlang config is valid!
```

- Browser: Chrome
- Operating system: Windows 11
- Screenshots ?

**Comment by ivanhofer** [comment]
We are aware of this issue and currently working on a refactor of our code base that will fix this issue: https://github.com/inlang/inlang/issues/797

**Comment by samuelstroschein** [comment]
@ivanhofer can you assign yourself to this issue to ensure this issue is handled?

**Comment by ivanhofer** [comment]
The issue should be fixed in version `0.8.0`
